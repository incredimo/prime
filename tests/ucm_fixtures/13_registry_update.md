{"add":{"name":"loggrep","version":"1.0.0","active":true,"tags":["log"]}}