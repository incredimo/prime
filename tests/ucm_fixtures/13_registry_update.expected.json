{
  "id": "sT3",
  "verb": "set",
  "attrs": { "path": ".prime/registry/index.json", "mode": "update" }
}